schedule/virt_autotest/slem-kubevirt-tests.yaml

```diff
@@ -0,0 +1,47 @@
+name:         slem-kubevirt-tests
+description:  >
+    Maintainer: Nan Zhang <nan.zhang@suse.com> qe-virt@suse.de
+    Run kubevirt tests on SLE Micro
+vars:
+    VIDEOMODE: text
+    SYSTEM_ROLE: kvm
+    OPT_KERNEL_PARAMS: hugepagesz=2M hugepages=64 max_loop=256
+schedule:
+    - '{{barrier_setup}}'
+    - '{{bootup_and_install}}'
+    - '{{kubevirt_tests}}'
+conditional_schedule:
+    barrier_setup:
+        TEST_SUITE_NAME:
+            slem-kubevirt-tests-server:
+                - virt_autotest/kubevirt_barriers
+    bootup_and_install:
+        WITH_SLEM_INSTALL:
```

Comment:
you have different yaml files to schedule SLE and SLEM tests, I guess it is because of different host installation steps. It is acceptable but you will have to maintain two files in the future.

Reply:
Updated to use one yaml file. Thank you for suggestion.